internal/services/items-service.go

```diff
@@ -0,0 +1,68 @@
+package services
+
+import (
+	"github.com/afmireski/garchop-api/internal/ports"
+	"github.com/afmireski/garchop-api/internal/validators"
+
+	customErrors "github.com/afmireski/garchop-api/internal/errors"
+	myTypes "github.com/afmireski/garchop-api/internal/types"
+)
+
+type ItemsService struct {
+	itemsRepository ports.ItemsRepositoryPort
+	cartsRepository ports.CartsRepositoryPort
+}
+
+func NewItemsService(itemsRepository ports.ItemsRepositoryPort, cartsRepository ports.CartsRepositoryPort) *ItemsService {
+	return &ItemsService{
+		itemsRepository: itemsRepository,
+		cartsRepository: cartsRepository,
+	}
+}
+
+func (s *ItemsService) RemoveItemFromCart(input myTypes.RemoveItemFromCartInput) *customErrors.InternalError {
+	if !validators.IsValidUuid(input.ItemId) {
+		return customErrors.NewInternalError("invalid user_id", 400, []string{"the item_id must be a valid uuid"})
+	}
+
+	if !validators.IsValidUuid(input.CartId) {
+		return customErrors.NewInternalError("invalid item_id", 400, []string{"the cart_id must be a valid uuid"})
+	}
```

Comment:
Inconsistência com os nomes dos campos.
```suggestion
	if !validators.IsValidUuid(input.ItemId) {
		return customErrors.NewInternalError("invalid item_id", 400, []string{"the item_id must be a valid uuid"})
	}

	if !validators.IsValidUuid(input.CartId) {
		return customErrors.NewInternalError("invalid cart_id", 400, []string{"the cart_id must be a valid uuid"})
	}
```